site/en/docs/android/custom-tabs/guide-interactivity/index.md

```diff
@@ -0,0 +1,197 @@
+
+```java
+CustomTabsIntent intent = new CustomTabsIntent.Builder()
+        ...
+        .addMenuItem("Menu item 1", pendingIntent)
+        .addMenuItem("Menu item 2", pendingIntent)
+        .addMenuItem("Menu item 3", pendingIntent)
+        .addMenuItem("Menu item 4", pendingIntent)
+        .addMenuItem("Menu item 5", pendingIntent)
+        .build();
+```
+
+## Customize the close button
+
+Customizing the close button can be a meaningful way to better fit a Custom Tab into the flow of your app. If you want the user to feel like Custom Tabs is a modal dialog, use the default `“X”` button. If you want the user to feel the Custom Tab is part of the application flow, use the back arrow.
+
+```java
+CustomTabsIntent.Builder intentBuilder = new CustomTabsIntent.Builder();
+intentBuilder.setCloseButtonIcon(BitmapFactory.decodeResource(
+    getResources(), R.drawable.ic_arrow_back));
+```
+
+## Add a bottom toolbar
+
+The bottom toolbar is a very flexible way to add more functionality to a custom tab. 
+
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/e0evUtBxkP0uI7kVj8PS.png", alt="Custom Tab with a bottom toolbar", width="320", height="693", class="screenshot screenshot--filled" %}
+
+By passing a [RemoteViews](https://developer.android.com/reference/android/widget/RemoteViews?cmdf=android+remoteviews) object to [CustomTabIntent.Builder#setSecondaryToolbarViews](https://developer.android.com/reference/android/support/customtabs/CustomTabsIntent.Builder.html#setSecondaryToolbarViews(android.widget.RemoteViews,%20int[],%20android.app.PendingIntent)), the bottom toolbar can be fully customized and dynamically updated.
+
+First we need to declare our toolbar layout. Create a new layout file `res/layout/custom_tab_toolbar.xml`:
+
+```xml
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:gravity="center">
+
+    <Button xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/ct_toolbar_next"
+        android:layout_width="wrap_content"
+        android:layout_height="48dp"
+        android:layout_margin="8dp"
+        android:padding="8dp"
+        android:paddingStart="16dp"
+        android:paddingEnd="16dp"
+        android:text="Next" />
+
+    <Button xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/ct_toolbar_previous"
+        android:layout_width="wrap_content"
+        android:layout_height="48dp"
+        android:layout_margin="8dp"
+        android:padding="8dp"
+        android:text="Previous" />
+</LinearLayout>
+```
+
+{% Aside 'gotchas' %}
+The Custom Tab color scheme does not apply here, you need to manually style the toolbar. If your app provides a light and a dark scheme, you need to configure two different toolbars, one for each color scheme.
```

Comment:
is scheme the normal word? without color, scheme reads weird. I would default to `theme` if I was trying to use a single word.

Reply:
that's what the custom tabs API is called, added a link to https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder?cmdf=custom%20tabs%20intent%20builder#setDefaultColorSchemeParams(androidx.browser.customtabs.CustomTabColorSchemeParams)